src/fileformat/types/dotnet_headers/blob_stream.cpp

```diff
@@ -25,22 +22,55 @@ BlobStream::BlobStream(std::uint64_t streamOffset, std::uint64_t streamSize) : S
  */
 std::vector<std::uint8_t> BlobStream::getElement(std::size_t offset) const
 {
-	auto itr = elements.find(offset);
-	if (itr == elements.end())
+	// Ugly, C like just for fast prototype, rework TODO
+	std::uint32_t len = 0;
+	const unsigned char* ptr = data.data() + offset;
+	if (offset >= data.size())
+	{
 		return {};
+	}
+	else if ((*ptr & 0x80) == 0x00)
+	{
+		len = *ptr;
+		offset += 1;
+		if (offset + len <= data.size())
```

Comment:
I think it's okay, because in C++ iterators end iterators is one element past the collection. So if offset = 0, len = 2 and we have data.size() = 2; then 0 + 2 <= 2, so we return the 2 elements of data.